plugins/base/src/main/resources/dokka/scripts/symbol-parameters-wrapper_deferred.js

```diff
@@ -0,0 +1,63 @@
+// helps with some corner cases where <wbr> starts working already,
+// but the signature is not yet long enough to be wrapped
+const leftPaddingPx = 60
+
+const wrapAllSymbolParameters = () => {
+    document.querySelectorAll("div.symbol").forEach(symbol => wrapSymbolParameters(symbol))
+}
+
+const wrapSymbolParameters = (symbol) => {
+    let symbolBlockWidth = symbol.clientWidth
+    let innerTextWidth = Array.from(symbol.children)
+        .map(it => it.getBoundingClientRect().width).reduce((a, b) => a + b, 0)
+
+    // if signature text takes up more than a single line, wrap params for readability
+    let shouldWrapParams = innerTextWidth > (symbolBlockWidth - leftPaddingPx)
+    if (shouldWrapParams) {
+        let parameters = symbol.querySelector("span.parameters")
+        if (parameters != null) {
+            parameters.classList.add("wrapped")
+            parameters.querySelectorAll("span.parameter").forEach(param => {
+                // has to be a physical indent so that it can be copied. styles like
+                // paddings and `::before { content: "    " }` do not work for that
```

Comment:
https://stackoverflow.com/questions/16552397/css-how-to-add-white-space-before-elements-content

It can improve performance.

Reply:
I tried JS fiddle from the answer, but when you select the next - it's not selecting spaces and thus not copying it to clipboard (it will copy the text without spaces), which is the biggest problem :( 

I'll give it a shot, but it doesn't look like it's going to work

![2022-09-12_17-32-33](https://user-images.githubusercontent.com/22685910/189695355-e38fb617-31af-45fc-8cef-d68f2f2fec1b.png)

Reply:
Yeah, no luck:
![2022-09-14_16-47-29](https://user-images.githubusercontent.com/22685910/190187533-c88c8e7a-9538-4b60-b2b3-aa0c112e8e5f.png)

And here's if I copy (Ctrl+C) it: 
```kotlin
fun <T> CoroutineScope.async(
context: CoroutineContext = EmptyCoroutineContext, 
start: CoroutineStart = CoroutineStart.DEFAULT, 
block: suspend CoroutineScope.() -> T
): Deferred<T>
```